hazelcast-sql/src/test/java/com/hazelcast/it/CheckSqlNoticeFileIT.java

```diff
@@ -1,27 +0,0 @@
-/*
- * Copyright 2021 Hazelcast Inc.
- *
- * Licensed under the Hazelcast Community License (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://hazelcast.com/hazelcast-community-license
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.hazelcast.it;
-
-import com.hazelcast.sql.impl.calcite.HazelcastSqlBackend;
-
-public class CheckSqlNoticeFileIT extends CheckNoticeFileIT {
```

Comment:
Why is it removed?

Reply:
The test should detect `HazelcastSqlBackend.class` which was removed in this PR.

Reply:
Use `JetSqlBackend` then?